components/compliance-service/api/tests/run.rb

```diff
@@ -105,6 +106,10 @@ def assert_grpc_error(details, code)
   def assert_equal_json_content(expected, actual)
     assert_equal JSON.parse(expected.to_json), JSON.parse(actual.to_json)
   end
+
+  def assert_equal_json_sorted(expected, actual)
+    assert_equal JSON.parse(expected).deep_sort, JSON.parse(actual).deep_sort
+  end
```

Comment:
If there are tests that rely on ordering, should we order the test data explicitly?

Reply:
```
    expected_job1 = {
      "id": job_id1,
      "name": "my Detect Job For two nodes",
      "type": "detect",
      "timeout": 600,
      "tags": [
        {
          "key": "reason",
          "value": "checking the connectivity"
        },
        {
          "key": "team",
          "value": "Spain"
        }
      ],
```
In the jobs output, tags(see example above) came with a random order, hence the deep sort of the json.